src/apps/backend/modules/shared-tasks/types.ts

```diff
@@ -0,0 +1,37 @@
+import { ApplicationError } from '../application';
+import { HttpStatusCodes } from '../http';
+
+export class SharedTask {
+  id: string;
+  task: any;
+  account: any;
```

Comment:
Avoid the use of type any. Use specific types for better safety.
task: string | Task;
account: string | Account;